content/client-lib-development-guide/features.textile

```diff
@@ -1361,15 +1370,42 @@ h4. ChannelOccupancy
 
 h4. ChannelMetrics
 
-* @(CHM1)@ @ChannelMetrics@ is a type that contains the count of publishers and subscribers, connections and presenceConnections, presenceMembers and presenceSubscribers 
+* @(CHM1)@ @ChannelMetrics@ is a type that contains the count of publishers and subscribers, connections and presenceConnections, presenceMembers and presenceSubscribers
 * @(CHM2)@ The attributes of @ChannelMetrics@ consist of:
 ** @(CHM2a)@ @connections@ integer - the total number of connections to the channel
 ** @(CHM2b)@ @presenceConnections@ integer - the total number of presence connections to the channel
 ** @(CHM2c)@ @presenceMembers@ integer - the total number of presence members for the channel
 ** @(CHM2d)@ @presenceSubscribers@ integer - the total number of presence subscribers for the channel
-** @(CHM2e)@ @publishers@ integer - the total number of publishers to the channel 
+** @(CHM2e)@ @publishers@ integer - the total number of publishers to the channel
 ** @(CHM2f)@ @subscribers@ integer - the total number of subscribers to the channel
 
+h4. BatchResult
+* @(BPA1)@ Contains the results from the batch operation
+* @(BPA2)@ @BatchResult@ has the following attributes:
```

Comment:
Idiomatic Go would return a pointer to a result (nil if no result) and and error. 
Like ` result, err := REST.batch.publish()` 
So I'm not sure if we should be prescribing that BatchResult has an error nested inside it. Maybe we can leave this open to being implemented idiomatically by each language.

Reply:
The issue here is that when a partial response is received, an error is supplied. Having an error response and a success response is impossible or discouraged in a lot of languages, perhaps either the partial error code should be ignored or full errors should be handled separately in a language-specific way?